pybricksdev/cli/__init__.py

```diff
@@ -311,6 +314,15 @@ async def reconnect_hub():
                             with _get_script_path(args.file) as script_path:
                                 await hub.download(script_path)
 
+                        case ResponseOptions.RUN_STORED:
+                            if hub.fw_version < Version("v1.2.0"):
```

Comment:
It's not actually the firmware version, but rather the Pybricks profile version that matters.